src/featbin/Makefile

```diff
@@ -15,7 +15,7 @@ BINFILES = compute-mfcc-feats compute-plp-feats compute-fbank-feats \
     process-kaldi-pitch-feats compare-feats wav-to-duration add-deltas-sdc \
     compute-and-process-kaldi-pitch-feats modify-cmvn-stats wav-copy \
     wav-reverberate append-vector-to-feats shift-feats concat-feats \
-	append-post-to-feats post-to-feats
+    append-post-to-feats post-to-feats
```

Comment:
As you are fixing the formatting of the file, could you please also sort the guys alphabetically? The list has become huge and hard to read.